### PR TITLE
Release/1.2.0

### LINKED ISSUE #110: Release version 1.2.0
### Describe your question

This issue is for tracking changes for the 1.2.0 release. Target release date: **June 2023**.

## [Release steps](https://github.com/10up/convert-to-blocks/blob/develop/CONTRIBUTING.md#release-instructions)

- [x] Branch: Starting from `develop`, cut a release branch named `release/1.2.0` for your changes.
- [x] Version bump: Bump the version number in `config.php`, `convert-to-blocks.php`, `readme.txt`, `package.json` and `package-lock.json` if it does not already reflect the version being released.  Update both the plugin "Version:" property in `convert-to-blocks.php` and the plugin `$plugin_version` constant in `config.php`.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary.  `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk`, ensuring you pull the most recent changes into `develop` first (`git checkout develop && git pull origin develop && git checkout trunk && git merge --no-ff develop`). `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] [Compare](https://github.com/10up/convert-to-blocks/compare/trunk...develop) trunk to develop to ensure no additional changes were missed.
- [ ] Test the pre-release ZIP locally by [downloading](https://github.com/10up/convert-to-blocks/actions/workflows/build-release-zip.yml) it from the Build release zip action artifact and installing it locally. Ensure this zip has all the files we expect, that it installs and activates correctly and that all basic functionality is working
- [x] Release: Create a [new release](https://github.com/10up/convert-to-blocks/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the [closed issues on the milestone](https://github.com/10up/convert-to-blocks/milestone/6?closed=1).  The release should now appear under [releases](https://github.com/10up/convert-to-blocks/releases).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/convert-to-blocks/actions) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/convert-to-blocks/.  This may take a few minutes.
- [x] Close the milestone: Edit the [milestone](https://github.com/10up/convert-to-blocks/milestone/6) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [ ] Punt incomplete items: If any open issues or PRs which were milestoned for `1.2.0` do not make it into the release, update their milestone to `1.2.1`, or `Future Release`

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by jeffpaul**
### Description of the Change
Release and version bump to 1.2.0.

Closes #110.

### How to test the Change
Verify manually via VS Code or the GitHub UI.

### Checklist:

- [x] I agree to follow this project's [**Code of Conduct**](https://github.com/10up/.github/blob/trunk/CODE_OF_CONDUCT.md).
- [x] I have updated the documentation accordingly.
- [ ] I have added tests to cover my change.
- [ ] All new and existing tests pass.

### CODE REVIEW

**Comment by jeffpaul** [comment]
Note that there's a discrepancy between https://github.com/10up/convert-to-blocks/pull/66 && https://github.com/10up/convert-to-blocks/pull/104 that we need to resolve before continuing with this release.

File: convert-to-blocks.php
Comment (jeffpaul) [inline convert-to-blocks.php:1]: @Sidsector9 what was the PR/change that necessitated the WP/PHP minimum bumps here?

Comment (jeffpaul) [reply convert-to-blocks.php:1]: #106

**Review by Sidsector9 — DISMISSED** [review]

**Comment by jeffpaul** [comment]
Updated release date, so otherwise proceeding with the earlier release approval

**Comment by jeffpaul** [comment]
Oh for the love of all things automated and holy, now a failing E2E test on WP trunk.  Looking at the string of commits, there's nothing from when that test was passing previously until now that should cause that failure here within the plugin repo.  My gut is that something wonky is going on in WordPress 6.3 (in fact there's a [bit of a hiccup in the Beta 1 release](https://make.wordpress.org/core/2023/06/27/wordpress-6-3-beta-1-postponed/)) so I'm choosing to chalk it up to oddities there that we'll catch & update when it gets to testing for WP 6.3 compatibility / "tested up to".

**Pull request merged**